### PR TITLE
Adjust SkiaSharp loading

### PR DESCRIPTION
**Pull request by jeromelaban**
fixes https://github.com/unoplatform/uno/issues/17897

This PR avoids nuget-cache file copy to match the current runtime and underlying platform, and loads skiasharp in place.

### CODE REVIEW

**Review by ajpinedam — APPROVED** [review]

File: src/Resizetizer/src/SkiaSharpTools.Initializer.cs [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            }
+        }
+    }
+
+    private static void SetupResolver()
+    {
+        var dllImportResolverDelegateType = Type.GetType("System.Runtime.InteropServices.DllImportResolver")!;
+
+        // We're building with netstandard 2.0 which does not provide those APIs, but we
+        // know we're running on netcore. Let's use them through reflection.
```
Comment (Youssef1313) [inline src/Resizetizer/src/SkiaSharpTools.Initializer.cs:62]: Are we sure we're running on .NET Core? Was this tested when building from Visual Studio and not only from command-line?

Comment (Youssef1313) [reply src/Resizetizer/src/SkiaSharpTools.Initializer.cs:62]: Oh I see the conditional call. Didn't notice it earlier. That's good then

File: src/Resizetizer/src/SkiaSharpTools.Initializer.cs [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    private static bool _initialized;
+    private static MethodInfo _setDllImportResolver;
+    private static TryLoadDelegate _tryLoad;
+
+    static SkiaSharpTools()
+    {
+        Initialize();
+    }
+
+    public static void Initialize()
```
Comment (Youssef1313) [inline src/Resizetizer/src/SkiaSharpTools.Initializer.cs:25]: As it's called only from the static constructor, I think could be made private

File: src/Resizetizer/src/SkiaSharpTools.Initializer.cs [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    {
+        if (!_initialized)
```
Comment (Youssef1313) [inline src/Resizetizer/src/SkiaSharpTools.Initializer.cs:27]: Is it possible to get to this twice? It's only called from static constructor I think.

File: src/Resizetizer/src/SkiaSharpTools.Initializer.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Resizetizer/src/SkiaSharpTools.Initializer.cs:62]: msbuild is the same as VS in this context, it's running on .NET Framework.

File: src/Resizetizer/src/SkiaSharpTools.Initializer.cs (hunk 3 above)
Comment (jeromelaban) [reply src/Resizetizer/src/SkiaSharpTools.Initializer.cs:27]: It's there to be called if another class wants to use SkiaSharp outside of SkiaSharpTools.

**Pull request merged**